curvefs/src/mds/fs_manager.cpp

```diff
@@ -413,7 +451,24 @@ FSStatusCode FsManager::CreateFs(const ::curvefs::mds::CreateFsRequest* request,
             LOG(ERROR) << "CreateFs fail, insert trash dentry fail"
                     << ", fsName = " << fsName
                     << ", ret = " << FSStatusCode_Name(ret);
-            // TODO(chenwei): delete fs
+            // delete partition and fs
```

Comment:
delete recyle inode
delete root inode
delele partition
delete fs

at this order